src/app/components/HomePageNonCritical.tsx

```diff
@@ -1,72 +1,181 @@
 import { Tile, HeroContainer } from 'franklin-sites';
 
-import Block from './Block';
+import colors from '../../../node_modules/franklin-sites/src/styles/colours.json';
+
+// Namespaces
+import UniProtKBIllustration from '../../svg/uniprotkb_illustration.svg';
```

Comment:
All of those I would probably prefer to load them as actual files. Here we are using them as images so we don't have any advantage bundling them inside react code.
Loading them as files would:
 - improve caching (at the cost of more different requests)
 - reduce javascript payload (quicker to download and to process)
 - make the image rendering non-blocking for the page (can appear later, it's fine)
Either make it specific per-import by doing something like `import uniProtKBIllustrationURL from '!file-loader!../../svg/uniprotkb_illustration.svg';` or change the webpack config to have all the SVGs inside that `svg` folder use `file-loader`.
In any case, usage would be as `backgroundImage={<img src={UniProtKBIllustration} />}`

Reply:
TRM-25295